src/views/OtherTools.md

```diff
@@ -6,21 +6,33 @@
         <p>
                 <a href="http://monarch-exomiser-web-dev.monarchinitiative.org/exomiser">Exomiser</a>
                 is a Java program that functionally annotates variants
-                from whole-exome sequencing data starting from a VCF file (version 4).
-                The functional annotation code is based on Jannovar and uses UCSC
-                KnownGene transcript definitions and hg19 genomic coordinates.<br>
-                Variants are prioritized according to user-defined criteria on variant
-                frequency, pathogenicity, quality, inheritance pattern, and model
-                organism phenotype data. Predicted pathogenicity data was extracted
+                from whole-exome sequencing data starting from a VCF file (version 4) and a set 
+                of phenotypes encoded using the <a href="https://hpo.jax.org/app/">Human Phenotype 
+                Ontology</a>. The functional annotation of variants is based on Jannovar and uses UCSC
+                KnownGene or RefSeq transcript definitions and hg19 or hg38 genomic coordinates. Variants 
+                are prioritized according to user-defined criteria on variant frequency, pathogenicity, 
+                quality, inheritance pattern, and model organism phenotype data. Predicted pathogenicity data is extracted
                 from the dbNSFP resource. Cross-species phenotype comparisons are
-                powered by the <a href="http://owlsim.org">OWLSim algorithm</a>.<br>
-                Exomiser is currently using mouse phenotypes and will soon be leveraging
-                the zebrafish phenotype data. Worm and fly phenotype data will be
-                available later this year.<br>
-                Exomiser is available from the Sanger Institute. A paper describing Exomiser
-                is available <a href="http://genome.cshlp.org/content/24/2/340.long">here</a>.
+                powered by the <a href="http://owlsim.org">OWLSim algorithm</a> and use human, mouse, 
+                and fish phenotypes. Exomiser is available for download <a href="https://github.com/exomiser/Exomiser"> 
+                here</a>. A paper describing 
+                Exomiser is available <a href="http://genome.cshlp.org/content/24/2/340.long">here</a>.
         </p>
     </div>
+    <div class="card">
+            <h4>LIRICAL</h4>
+            <p>
+                <a href="https://lirical.readthedocs.io/en/latest/">LIRICAL</a>
+                is Java software that provides prioritization of candidate diseases using a likelihood 
+                ratio statistical framework. LIRICAL takes as input a set of phenotypic abnormalities
+                encoded using the <a href="https://hpo.jax.org/app/">Human Phenotype Ontology</a>. 
```

Comment:
@justaddcoffee - a super tiny typo here. 
Either replace period with a comma after "Human Phenotype Ontology" in line 28. Else, change the word 'and' in line 29 to start with upper case 'A.'

Reply:
Done, thanks for the sharp eyes Moni